demos/gesture_recognition_demo/python/README.md

```diff
@@ -102,7 +103,11 @@ python gesture_recognition_demo.py \
 
 The demo starts in person tracking mode and to switch it in the action recognition mode you should press `0-9` button with appropriate detection ID (the number in top-left of each bounding box). If frame contains only one person, they will be chosen automatically. After that you can switch back to tracking mode by pressing space button.
 
-An example of file with class names can be found within the OMZ directory at `<omz_dir>/data/dataset_classes/msasl100.json`.
+An example of file with class names can be found within the OMZ directory:
+
+* MS-ASL-100: `<omz_dir>/data/dataset_classes/msasl100.json`
+* Jester-27: `<omz_dir>/data/dataset_classes/jester27.json`
+* Common-Sign-Language-12: `<omz_dir>/data/dataset_classes/common_sign_language.json`
```

Comment:
I think it makes sense to add number of classes to file name for avoid conflicts in the future if will be added models with different number of supported classes

Reply:
Done